app/views/spree/admin/shared/_order_tabs.html.haml

```diff
@@ -17,8 +17,9 @@
       %dt#order_status
         = t(:status)
       %dd
-        %span{:class => "state #{@order.state}"}
-        = t(@order.state, scope: :order_state)
+        - span_classes = "state #{@order.state}"
```

Comment:
can you pls rename this to order_state_classes and the next two ones to shipment_state_classes and payment_state_classes?

Reply:
hashtag #sorryforbeingannoyingonyourpr

Reply:
All good !